ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,175 @@
+More information about Ansible inventories here.
+
+### Playbooks
+Now that the Ansible inventory is defined, we can tell Ansible what to do on the switches.
+This is defined in one or more ***playbooks*** as they’re called in Ansible-lingo.
+A ***playbook***, like an inventory file, is a text file written using YAML format and it contains one or more ***plays***, that group a set of instructions to be executed in an ordered list. 
+The terms ***playbook*** and ***play*** are sports analogies. Each play executes part of the overall goal of the playbook, running one or more ***tasks***. Each ***task*** calls an Ansible ***role*** or ***module***, such as ***ansible.avd.eos_cli_config_gen***. 
+A ***playbook*** runs in order from top to bottom. Within each ***play***, ***tasks*** also run in order from top to bottom. ***Playbooks*** with multiple ***plays*** can orchestrate multi-machine deployments, running one ***play*** on your Ansible development machine, another on your CV instance, and another on each EOS device you look to configure. 
+At a minimum, each ***play*** defines two things:
+1. The managed devices (***hosts***) to target, referenced from the ***inventory*** we defined earlier
+2. One or more ***tasks*** to execute on the targets defined
+
+The ***hosts*** specified in a ***playbook*** references variables defined in the ***inventory***. From a ***playbook*** you can select large or small groups of the ***inventory***, right down to individual ***hosts*** to be as specific as possible with any configuration changes.
+
+A simple example of a ***play*** defined in a ***playbook*** could be the following:
+
+```
+# Play to build EOS configuration from EOS_DESIGNS
+- hosts: DC1_FABRIC
+  tasks:
+    # BUILD EOS configuration
+    - name: generate intended variables
+      import_role:
+         name: arista.avd.eos_designs
+    - name: generate device intended config and documentation
+      import_role:
+         name: arista.avd.eos_cli_config_gen
+```
+
+Reading from the top, the ***hosts*** references the variable ***DC1_FABRIC***, which is defined in the ***inventory*** as all ***hosts*** further defined within DC1. In an actual deployment, this would likely include groups of devices, such as ***DC1_SPINES***, ***DC1_L3LEAFS*** and so on - again naming convention used here is just an example, not set in stone.
+The ***tasks*** are then defined and the first task imports a ***role*** called ***arista.avd.eos_designs***. A ***role*** is where the majority of things done with Ansible happen. In the ***arista.avd.eos_designs*** role, we generate the variables used for further processing in subsequent roles, such as the second one shown above called ***arista.avd.eos_cli_config_gen***. This role generates the intended configuration along with documentation in markdown format.
```

Comment:
We should talk about inputs and outputs per role, and possibility for the user to use only some of the roles, like only cli_config_gen.